ydb/src/client_topic/create_topic_options.rs

```diff
@@ -0,0 +1,37 @@
+use crate::grpc_wrapper::raw_topic_service::common::codecs::RawSupportedCodecs;
+use crate::grpc_wrapper::raw_topic_service::common::consumer::RawConsumer;
+use crate::grpc_wrapper::raw_topic_service::common::metering_mode::RawMeteringMode;
+use derive_builder::Builder;
+use std::collections::HashMap;
+use crate::errors;
+use crate::YdbError::{InternalError};
+
+#[derive(Builder)]
+pub struct TopicOptions { // Use TopicOptionsBuilder
```

Comment:
It will be better move options to client.rs file - near with method.

It should have reason for separate it.
In go options separated for workarond of type/method visibility restrictions.
In rust crate most options placed near the method.